openff/arsenic/arsenic.py

```diff
@@ -1,5 +1,5 @@
 # this will combine a single command that will run all analysis and save everything
-from . import plotting, wrangle
+from openff.arsenic import plotting, wrangle
```

Comment:
Why the change here?

Reply:
The PEP8 [recommendation](https://www.python.org/dev/peps/pep-0008/#imports) is to use absolute imports, but I guess both should work.

Reply:
That's what I thought, let's use absolute imports!